### PR TITLE
Update 2019-07-uccs-research-project.md

### PR DESCRIPTION
**Pull request by biophil**
Formatting and minor language changes

### CODE REVIEW

File: _workers/2019-07-uccs-research-project.md [hunk 1]
```diff
@@ -27,15 +27,15 @@ This worker proposal, if approved and funded, will support a gift to the Univers
 
 By far the most critical budget items are the stipend and tuition payments for the PhD student, who will be committing the bulk of the time to the project. In addition to the student’s budget items, the project’s impact is significantly enhanced by including some time for Dr. Brown, to ensure that he can adequately direct the research. Time for faculty can be included as salary in the summer months or as a “course buy-out” during the academic year which reduces his teaching load to allow more time for research. The following budget includes a summer month as well as a teaching reduction. Finally, an annual $10,000 travel budget is included; both Dr. Brown and the PhD student will travel to universities, conferences, and workshops to promote the work, disseminate research findings and develop future collaborative relationships with other researchers. Each of the following items includes a 10% “tax” which is taken by the University to cover facilities maintenance and support staff salaries.
```
Comment (ryanRfox) [inline _workers/2019-07-uccs-research-project.md]: I prefer to eliminate the use of the term "tax" in favor of a more neutral/academic sounding term "indirect costs". Perhaps the final sentence reads, "UCCS applies indirect costs of 10% to each budget line item below to cover facilities maintenance, support staff salaries and related grant management expenses."

File: _workers/2019-07-uccs-research-project.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ category: Research
 bfid: 201907-uccs-research-project
 workerid:
 name: University of Colorado at Colorado Springs
-title: "BitAsset PhD Research grant - University of Colorado"
+title: "BitAsset PhD Research project - University of Colorado"
 name: Philip N. Brown, Assistant Professor of Computer Science
```
Comment (ryanRfox) [inline _workers/2019-07-uccs-research-project.md]: Please add the "Dr." title to the name filed.

File: _workers/2019-07-uccs-research-project.md [hunk 3]
```diff
@@ -55,28 +55,28 @@ Prof. Philip Brown wrote an executive summary that contains the full details of
 Read [full executive summary](http://cs.uccs.edu/~pbrown2/files/BTS_executive_summary.pdf)
```
Comment (ryanRfox) [inline _workers/2019-07-uccs-research-project.md]: Please change title to "Dr." from "Prof."

File: _workers/2019-07-uccs-research-project.md [hunk 4]
```diff
@@ -55,28 +55,28 @@ Prof. Philip Brown wrote an executive summary that contains the full details of
...
+* Develop a “science of collateralized tokens,” investigating the key design tradeoffs in the space.
+* Can the balance between short-term and long-term stability be tuned?
+* Is there a fundamental tradeoff in collateralized tokens between manipulability and short-term stability?
+* How can we assess risk in collateralized token systems, taking into account the incentives seen by all parties?
 
 **Additional project activities**:
-Travel to universities and academic/blockchain conferences to present our research.
-Participation and outreach in blockchain community events in the greater Colorado Springs area.
+* Travel to universities and academic/blockchain conferences to present our research.
+* Participation and outreach in blockchain community events in the greater Colorado Springs area.
```
Comment (ryanRfox) [inline _workers/2019-07-uccs-research-project.md]: I will appreciate mention of sharing ongoing research findings with Core Team and contributing insights to relevant design discussions.

File: _workers/2019-07-uccs-research-project.md (hunk 1 above)
Comment (biophil) [reply _workers/2019-07-uccs-research-project.md]: good idea -- added your suggested language.

File: _workers/2019-07-uccs-research-project.md (hunk 4 above)
Comment (biophil) [reply _workers/2019-07-uccs-research-project.md]: Included.

**Pull request merged**